### DISCUSSION TITLE
google chrome dev not seeing my security onion SOC, but google chrome regular does?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10786] [Unsupported Versions]
Hi, I used to be able to access my SOC with chrome developer, but all of a sudden it gives me the “this site cant be reached”. But it comes up just fine in regular google chrome. Thought I would see if anyone else has had this issue?
   I uninstalled and reinstalled it, same issue. 

Thanks for any advice or suggestions

**Comment by iqworks** [comment]
I wanted to share the error mgs.
![forum 000](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/64d87007-b916-45ab-b150-286ed6d01231)
![forum 001](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/fe567d16-c411-4750-b369-d49dd6616362)

**Comment by Ryan8bit** [comment]
Click advanced. Then click proceed to ...

**Comment by iqworks** [comment]
HI Ryan, I am sorry, I sent the WRONG PIC!!!

![forum 002](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/e5ed7e0b-afe3-4cea-b08f-b578e82122bc)

![forum 003](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/70e630a2-2e35-411d-be73-858b5e3ba66c)

(there is a detail tab, let me know if you need that?)
thanks for your time

**Comment by dougburks** [comment]
I just tested Google Chrome Dev against Security Onion 2.3.260 and it works fine for me. Here's my full Google Chrome version:
```
Version 117.0.5897.3 (Official Build) dev (64-bit)
```

If regular Google Chrome works for you, please use that.

**Reply by iqworks** [reply]
I will try with 2.3.260. thanks
